pkg/cvo/cvo.go

```diff
@@ -294,15 +303,21 @@ func loadConfigMapVerifierDataFromUpdate(update *payload.Update, clientBuilder v
 		src := fmt.Sprintf("the config map %s/%s", manifest.Obj.GetNamespace(), manifest.Obj.GetName())
 		data, _, err := unstructured.NestedStringMap(manifest.Obj.Object, "data")
 		if err != nil {
-			return nil, errors.Wrapf(err, "%s is not valid: %v", src, err)
+			return nil, nil, errors.Wrapf(err, "%s is not valid: %v", src, err)
 		}
 		verifier, err := verify.NewFromConfigMap(src, data, clientBuilder)
 		if err != nil {
-			return nil, err
+			return nil, nil, err
 		}
-		return verifier, nil
+
+		// allow the verifier to consult the cluster for signature data, and also configure
+		// a process that writes signatures back to that store
+		signatureStore := verifyconfigmap.NewStore(configMapClient, nil)
+		verifier = verifier.WithStores(signatureStore)
+		persister := verify.NewSignatureStorePersister(signatureStore, verifier)
```

Comment:
Having `signatureStore` baked into `verifier` and also living as a sibling here feels strange, but I'm still wrapping my head around these new functions.